src/components/molecules/Visualizer/Engine/Cesium/Box/index.tsx

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Can you use `// eslint-disable-line react-hooks/exhaustive-deps` only where it is necessary instead of on the whole file? I feel like ignoring on the whole file is kinda dangerous

Reply:
I fixed in [chore: remove unnecessary disabling eslint comment](https://github.com/reearth/reearth-web/pull/338/commits/6f65b3b54d9e32cd1038fcdd466558aecc39c991) :pray: